lte/gateway/c/core/oai/tasks/nas/esm/esm_data_context.c

```diff
@@ -89,23 +96,16 @@ void nas_stop_T3489(esm_context_t* const esm_ctx) {
     ue_mm_context_t* ue_mm_context =
         PARENT_STRUCT(emm_context, struct ue_mm_context_s, emm_context);
     mme_ue_s1ap_id_t ue_id = ue_mm_context->mme_ue_s1ap_id;
-    void* nas_timer_callback_args;
-    esm_ctx->T3489.id =
-        nas_timer_stop(esm_ctx->T3489.id, (void**) &nas_timer_callback_args);
-    if (NAS_TIMER_INACTIVE_ID == esm_ctx->T3489.id) {
-      OAILOG_INFO(
-          LOG_NAS_EMM, "T3489 stopped UE " MME_UE_S1AP_ID_FMT "\n", ue_id);
-      if (nas_timer_callback_args) {
-        esm_ebr_timer_data_t* data =
-            (esm_ebr_timer_data_t*) nas_timer_callback_args;
-        data->ctx = NULL;
-        bdestroy_wrapper(&data->msg);
-        free_wrapper((void**) &data);
-      }
-    } else {
-      OAILOG_ERROR(
-          LOG_NAS_EMM, "Could not stop T3489 UE " MME_UE_S1AP_ID_FMT "\n",
-          ue_id);
+    mme_app_stop_timer(esm_ctx->T3489.id);
+    esm_ctx->T3489.id = NAS_TIMER_INACTIVE_ID;
+
+    OAILOG_DEBUG(
```

Comment:
Is the IMSI available here? Since ue_mm_context is retrieved here should we update this to `OAILOG_DEBUG_UE` to reduce the IMSI logs gap?

Reply:
let's handle the log changes in another PR for `OAILOG_DEBUG_UE`.